setup.py

```diff
@@ -1,66 +1,12 @@
-import re
 from collections.abc import Mapping
-from pathlib import Path
 from typing import cast
 
 from setuptools import Command, find_packages, setup
 
 import versioneer
 
-# Add README.md as long description
```

Comment:
Will these changes drop the description etc from pypi?

Reply:
I actually moved the descriptions in a different PR. They're now specified by the following two lines in pyproject.toml:
``
description = "xDSL"
readme = "README.md"
```
This diff just removes some dead code that I forgot to remove in the previous PR.